tests/api/test_grid_workflow.py

```diff
@@ -48,6 +48,14 @@ def test_gridworkflow():
     # test backend : that it finds the expected cell/dataset
     assert list(gw.cell_observations(**query).keys()) == [(1, -2)]
 
+    # again but with geopolygon
+    assert list(gw.cell_observations(**query,
+                                     geopolygon=gridspec.tile_geobox((1, -2)).extent).keys()) == [(1, -2)]
+
+    with pytest.raises(ValueError):
+        list(gw.cell_observations(**query, tile_buffer=(1, 1),
```

Comment:
Why is this an error? Probably worth mentioning

Reply:
Added code to indicate which exception is being tested. As to why it's not supported mode to have tile_buffer as well as geopolygon? Probably because this function is really three functions internally.